docs/platforms/rust/common/performance/index.mdx

```diff
@@ -23,7 +23,7 @@ The two options are meant to be mutually exclusive. If you set both, <PlatformId
 
 <PlatformContent includePath="performance/configure-sample-rate" />
 
-Learn more about performance monitoring <PlatformLink to="/configuration/options/#tracing-options">options</PlatformLink>, or how to <PlatformLink to="/configuration/sampling/#sampling-transaction-events">sample transactions</PlatformLink>.
+Learn more about performance monitoring <PlatformLink to="/configuration/options/#tracing-options">options</PlatformLink>, how to use the <PlatformLink to="/configuration/sampling/#setting-a-sampling-function">tracesSampler</PlatformLink> function, or how to <PlatformLink to="/configuration/sampling/#sampling-transaction-events">sample transactions</PlatformLink>.
```

Comment:
Looks like that Rust page doesn't have a "Setting a sampling function" header. Still goes to the right page, but not sure if you want to link to a different section?